Brief-over-of-Heads-and-firmware-security/Influences.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Influences and prior works
+permalink: /Influences/
+nav_order: 3
+parent: Brief overview of Heads and firmware security
+---
+
+Heads is influenced by several years of firmware vulnerability
+ research by Trammell Hudson including [Thunderstrike](https://trmm.net/Thunderstrike)
+ and [Thunderstrike 2](https://trmm.net/Thunderstrike_2)
+
+As well as many other researchers' work
+
+* "[Hardening hardware and choosing a #goodBIOS](https://media.ccc.de/v/30C3_-_5529_-_en_-_saal_2_-_201312271830_-_hardening_hardware_and_choosing_a_goodbios_-_peter_stuge#t=2372)
```

Comment:
Goodbios ref is here

Reply:
@Thrilleratplay https://github.com/osresearch/heads-wiki/pull/49/files/1899083cd24a471d83b67df360b4a174165d8bd3#diff-89e91d5fe0f3cf3e8289a4a683dd2108R15